src/odb/src/db/odb.tcl

```diff
@@ -465,6 +465,102 @@ proc report_voltage_domains {} {
   }
 }
 
+
+
+sta::define_cmd_args "report_cell_count" {}
+
+proc report_cell_count {} {
+  set db [ord::get_db]
+  set chip [$db getChip]
+  if { $chip == "NULL" } {
+    utl::error ODB 363 "please load the design before trying to use this command"
+  }
+  set block [$chip getBlock]
+  set insts [$block getInsts]
+  set fill "0"
+  set clock_inverter "0"
+  set clock_buffer "0"
+  set macro "0"
+  set pad "0"
+  set synth "0"
+  set ant "0"
+  set time_repr_inverter "0"
+  set time_repr_buffer "0"
+  
+  utl::report "\nReporting Cells:"
+
+  foreach inst $insts {
+    set master [$inst getMaster]
+    set master_type [$master getType]
+    set source_type [$inst getSourceType]
+    set cell_name [$master getName]
+    if { bool([$master isBlock]) == bool(true) } {
```

Comment:
can you write
```
    if { [$master isBlock] } {
```
?

Reply:
I don't think tcl supports this, but I can try.

Reply:
Yes, It works, I will change for the other cases as well.